app/Http/Livewire/CollectionPage.php

```diff
@@ -6,12 +6,16 @@
 use Livewire\Component;
 use Livewire\ComponentConcerns\PerformsRedirects;
 use Lunar\Models\Collection;
+use Livewire\WithPagination;
+use Lunar\Models\Url;
 
 class CollectionPage extends Component
 {
     use PerformsRedirects,
         FetchesUrls;
+    use WithPagination;
```

Comment:
Maybe format it this way to match the other imported traits?

```suggestion
    use PerformsRedirects,
        FetchesUrls,
        WithPagination;
```